src/App.js

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { FileUploader } from "react-drag-drop-files";
+import {
+  Button,
+  Container,
+  Form,
+  Modal,
+  Navbar,
+  OverlayTrigger,
+  Tooltip,
+} from "react-bootstrap";
+
+const fileTypes = ["jpg", "jpeg", "png", "bmp", "gif"];
+function App() {
+  const [token, setToken] = React.useState(null);
+  const [repository, setRepository] = React.useState(null);
+  const [show, setShow] = React.useState(false);
+  const [image, setImage] = React.useState(null);
+
+  return (
+    <>
+      <Navbar bg="dark" variant="dark">
+        <Container className="Nav">
+          <Navbar.Brand href="/">My Photohub 📸</Navbar.Brand>
+          <Navbar.Toggle />
+          <Navbar.Collapse className="justify-content-end">
+            <Navbar.Text>
+              {repository
```

Comment:
Here, you are checking if the repository exists but I think there should be `token` too. 
Something like;

```js
(token && repository)
```

Reply:
Done! 👍